[CUBRIDMAN-19] update jdbc_driver usage

kmjc6611 opened the pull request:
http://jira.cubrid.org/browse/CUBRIDMAN-19

kmjc6611 committed:
update jdbc_driver usage

kmjc6611 committed:
update URL for build

kmjc6611 committed:
restore python.rst

kmjc6611 commented:
I excluded changes about python.rst to make a new PR for python.rst.

hgryoo commented on en/api/jdbc.rst:
```diff
@@ -664,15 +664,15 @@ The line "a" in the example 1 is where data of collection types (**SET**, **MULT
-                   oid.removeFromSet("multisettest",new Integer(2));
-                   oid.removeFromSequence("listtest",99);
-                   oid.removeFromSequence("listtest",1);
+				   oid.addToSet("settest", Integer.valueOf(10));
+				   oid.addToSet("multisettest", Integer.valueOf(20));
+				   oid.addToSequence("listtest", 1, Integer.valueOf(30));
+				   oid.addToSequence("listtest", 100, Integer.valueOf(100));
+				   oid.putIntoSequence("listtest", 99, Integer.valueOf(99));
+				   oid.removeFromSet("settest", Integer.valueOf(1));
+				   oid.removeFromSet("multisettest", Integer.valueOf(2));
+				   oid.removeFromSequence("listtest", 99);
+				   oid.removeFromSequence("listtest", 1);
```
please fix the broken indentation in this example. you need to build the manual and check on a web browser after you revise.

kmjc6611 replied:
I check it and commit after build the manual.

hgryoo commented on ko/api/jdbc.rst:
```diff
@@ -667,15 +667,15 @@ OID를 사용할 때 다음의 규칙을 지켜야 한다.
-                   oid.removeFromSet("multisettest",new Integer(2));
-                   oid.removeFromSequence("listtest",99);
-                   oid.removeFromSequence("listtest",1);
+				   oid.addToSet("settest", Integer.valueOf(10));
+				   oid.addToSet("multisettest", Integer.valueOf(20));
+				   oid.addToSequence("listtest", 1, Integer.valueOf(30));
+				   oid.addToSequence("listtest", 100, Integer.valueOf(100));
+				   oid.putIntoSequence("listtest", 99, Integer.valueOf(99));
+				   oid.removeFromSet("settest", Integer.valueOf(1));
+				   oid.removeFromSet("multisettest", Integer.valueOf(2));
+				   oid.removeFromSequence("listtest", 99);
+				   oid.removeFromSequence("listtest", 1);
```
please fix the broken indentation in this example. you need to build the manual and check on a web browser after you revise.

kmjc6611 replied:
I check it and commit after build the manual.

kmjc6611 committed:
fix indentation

kmjc6611 committed:
Merge branch 'develop' into api-jdbc

kmjc6611 reviewed (commented):
I check it

kmjc6611 commented:
http://jira.cubrid.org/browse/CUBRIDMAN-19

Fixed wrong  contents in ko,en/jdbc.rst
Changed to latest version of CUBRID
Since Integrer is no longer used, i changed it to using valueOf
